SDCAlertView UI Tests/SDCAlertView_UI_Tests.swift

```diff
@@ -112,4 +125,16 @@ class SDCAlertView_UI_Tests: XCTestCase {
         showAlertAtIndex(8)
         XCTAssertTrue(XCUIApplication().activityIndicators["In progress"].exists)
     }
+
+    func testAlertWithAcessibilityIdentifiers() {
+        showAlertAtIndex(9)
```

Comment:
Style: please use `self.` here as well. I noticed I missed some of them too in some of the other cases - if you feel like fixing those that'd be great, otherwise I can do it separately.